src/main/java/Translator.java

```diff
@@ -105,7 +119,7 @@ protected void processLine(String line) {
         Scale scale = new Scale();
         for (int k = 0; k < itemCount; k++) {
             String label = row[k];
-            int value = Integer.parseInt(row[k + itemCount]);
+            CellContent value = new CellContent(row[k + itemCount]);
```

Comment:
Since it is peforming an integer division on the number of elements in the line to split it into two halves (i.e. it rounds down in the case of an uneven number of elements) I don't see any way of this ever indexing out of bounds.